ui/pages/login/index.js

```diff
@@ -20,31 +22,32 @@ function oidcLogin({ id, clientID, authURL, scopes }) {
   )}&state=${state}`
 }
 
-const KIND_OKTA = 'okta'
-
 function Providers({ providers }) {
   return (
     <>
       <div className='mt-2 w-full max-w-sm'>
-        {providers.map(p => (
-          <button
-            onClick={() => oidcLogin(p)}
-            key={p.id}
-            title={`${p.name} — ${p.url}`}
-            className='my-1.5 flex w-full flex-row items-center justify-center rounded-md border border-gray-700 px-4 py-3 hover:hover:border-gray-600'
-          >
-            <img
-              alt='identity provider icon'
-              className='h-4'
-              src={`/providers/${p.kind}.svg`}
-            />
-            {p.kind !== KIND_OKTA && (
-              <span className='relative pl-2 text-sm font-semibold leading-none'>
-                Single Sign-On
-              </span>
-            )}
-          </button>
-        ))}
+        {providers.map(
+          p =>
+            p.kind && (
+              <button
+                onClick={() => oidcLogin(p)}
+                key={p.id}
+                title={`${p.name} — ${p.url}`}
+                className='my-2 flex w-full items-center rounded-md border border-gray-700 px-4 py-3 hover:hover:border-gray-600'
+              >
+                <img
+                  alt='identity provider icon'
+                  className='h-4'
+                  src={`/providers/${p.kind}.svg`}
+                />
+                <span className='items-center pl-4 text-xs capitalize text-gray-300'>
```

Comment:
Last nit: we'd just want to capitalize `p.name` (and not `with`)

Great work with this!